app/assets/stylesheets/application.scss

```diff
@@ -26,3 +26,7 @@
 // import custom scss, content to be set in deployment.
 //
 @import "tail";
+
+.modal-footer form{
+  margin: 0;
+}
```

Comment:
The comment in this file should make it more clear, but nothing should come after `@import "tail"`. The purpose of importing tail.scss is to all someone deploying the app to override all other css, but they can't do this if there is css after the import.

the proper place to add css is in leap.scss